### ISSUE TITLE
collect + merge rss feeds from db

### ISSUE THREAD

**Issue by Godley** [csblogs/csblogs#5]
need to create a class or method which pulls all of the feed links from the database and merges them into a single feed

**Comment by Godley** [comment]
Planning on working on this when I get home if no one has any objections/working on it already

**Comment by emashliles** [comment]
Are you working on this still?

**Comment by Godley** [comment]
I didn't get very far on this before realising @DannyBrown has a very specific idea of what he thinks we should do for this, so I stopped. Still waiting on him to split it into further issues, but his view on this (from what I gather) is we should not only aggregate the feeds, but also store the new posts to our own db so that a person can have more than 10 or so blog posts displayed on the site at a time, as the old hullcsblogs did with the wordpress aggregator plugin. I'm not sure on the ethics of that as bloggers might not be comfortable with their posts being stored somewhere else on the internet, but meh.
@DannyBrown did I describe that correctly?
if you want to take a shot at doing this from a regular aggregator point of view please be my guest 'cause I think it's too big a task for my own knowledge of node.js at this stage, but the way everything's being handled needs confirming
